packages/snaps-ui/src/builder.test.ts

```diff
@@ -16,9 +16,10 @@ describe('copyable', () => {
       value: 'Hello, world!',
     });
 
-    expect(copyable({ value: 'foo bar' })).toStrictEqual({
+    expect(copyable({ value: 'foo bar', sensitive: true })).toStrictEqual({
```

Comment:
Can you add a case without, and this set to false too?

Reply:
There's a case without it right above ;) I'm adding a case with it set to `false`

Reply:
f8a8b13